sklearn/cluster/tests/test_k_medoids.py

```diff
@@ -0,0 +1,209 @@
+"""Testing for K-Medoids"""
+import numpy as np
+
+from sklearn.utils.testing import assert_equal
+from sklearn.utils.testing import assert_true
+from sklearn.utils.testing import assert_greater
+from sklearn.utils.testing import assert_array_equal
+from sklearn.utils.testing import raises
+
+from sklearn.metrics.pairwise import PAIRWISE_DISTANCE_FUNCTIONS
+from sklearn.metrics.pairwise import euclidean_distances
+from sklearn.datasets import load_iris
+
+from sklearn.cluster import KMedoids, KMeans
+
+from sklearn.utils.validation import NotFittedError
+
+
+def get_dummy_X():
+
+    np.random.seed(0)
```

Comment:
[Please avoid changing the seed.](http://scikit-learn.org/stable/developers/utilities.html#validation-tools)
Use instead:
```python
rng = np.random.RandomState(0)
X = rng.rand(100, 5)
```